spec/features/idv/steps/confirmation_step_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+feature 'idv confirmation step', :idv_job do
+  include IdvStepHelper
+
+  it_behaves_like 'idv confirmation step'
+
+  context 'with oidc' do
```

Comment:
I know this was copied, but in other tests we don't have the extra `context` blocks here.